src/components/auth/profile/MyRecurringCampaignsTable.tsx

```diff
@@ -0,0 +1,141 @@
+import { useTranslation } from 'next-i18next'
+import { useMutation } from '@tanstack/react-query'
+import { AlertStore } from 'stores/AlertStore'
+import { endpoints } from 'service/apiEndpoints'
+import { useSession } from 'next-auth/react'
+
+import { DataGrid, GridColDef, GridColumns, GridRenderCellParams } from '@mui/x-data-grid'
+import { IconButton, Tooltip, Box } from '@mui/material'
+import { money } from 'common/util/money'
+
+import CancelPresentationIcon from '@mui/icons-material/CancelPresentation'
+
+import { useGetUserRecurringDonations } from 'common/hooks/recurringDonation'
+
+import Link from 'components/common/Link'
+import { apiClient } from 'service/apiClient'
+import { authConfig } from 'service/restRequests'
+
+export default function MyRecurringCampaignsTable() {
+  const { t } = useTranslation()
+  const { data: recurringDonations } = useGetUserRecurringDonations()
+  const { data: session } = useSession()
+
+  const cancelMutation = useMutation({
+    mutationFn: async (id: string) => {
+      //TODO: this '/api/v1' does not look right here
+      apiClient.get(
+        endpoints.recurringDonation.cancelRecurringDonation(id).url,
+        authConfig(session?.accessToken),
+      )
+    },
+    onError: (err) => AlertStore.show(t('common:alerts.error') + err, 'error'),
```

Comment:
Have you checked what is the `err` param here. Is it okay to present to the user?

Reply:
fixed